src/metabase/public_settings.clj

```diff
@@ -926,8 +926,30 @@ See [fonts](../configuring-metabase/fonts.md).")
   :doc        false)
 
 (defsetting sql-parsing-enabled
-  (deferred-tru "SQL Parsing is disabled")
+  (deferred-tru "SQL Parsing is enabled. Please use [[query-analysis-native-disabled]] instead.")
+  :deprecated "0.50.14"
```

Comment:
are we 100% sure we want this backported?

Reply:
Not any more, the parent PR's backport had some conflicts so I abandoned it.